packages/tools/migrations-irec/deployment/migrations/canary/Dockerfile.local

```diff
@@ -0,0 +1,15 @@
+FROM energyweb/origin-backend-irec-app:canary
+
+RUN mkdir /var/migration-config
+COPY config/* /var/migration-config/
+
+WORKDIR /usr/local/share/.config/yarn/global/node_modules/\@energyweb/
+
+CMD ["/bin/bash","-c", "packages/apps/origin-backend-irec-app/deployment/packages/apps/origin-backend-irec-app/node_modules/.bin/typeorm migration:run --config exchange/dist/js/ormconfig.js && \
```

Comment:
Is this path correct? I don't think we install anything to `/usr/local/share/.config/yarn/global/node_modules/\@energyweb/`

Reply:
Would it be - 
for
- **migrations** - `WORKDIR /var/deployment/apps/origin-backend-app`
- **migrations-irec** - `WORKDIR /var/deployment/apps/origin-backend-irec-app`

Reply:
@kosecki123 I committed the changes. Can you please check and resolve the conflict.